authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -935,6 +980,17 @@ protected void removePolicyByMetadataObject(AuthorizationMetadataObject authzMet
     }
   }
 
+  protected String getConfValue(Map<String, String> conf, String key, String defaultValue) {
+    if (conf.containsKey(BaseCatalog.CATALOG_BYPASS_PREFIX + key)) {
+      return conf.get(BaseCatalog.CATALOG_BYPASS_PREFIX + key);
+    }
```

Comment:
Why do we need `BaseCatalog.CATALOG_BYPASS_PREFIX`?

Reply:
Because properties like `hadoop.security.xxx` should use PYPASS_PREFIX. It's the common behaviour in our system.

Reply:
Maybe `getByPassConfValue()` is a better function name?